app/src/components/createLearningJourney/CourseModal.jsx

```diff
@@ -23,7 +23,7 @@ export default function CourseModal({ skillId, coursesAndSkillsMapping, isModalO
 
   return (
     <div
-      className='fixed top-0 left-0 h-screen w-screen scale-100 backdrop-blur-3xl z-50'
+      className='fixed top-0 left-0 h-screen w-screen scale-100 backdrop-grayscale backdrop-blur-xl z-50'
```

Comment:
do you still want to include the grayscale

Reply:
I think its alright to leave it like this. The prev time where the grayscale was abit problematic was when it was used without the `backdrop` prefix